spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/SecretsConfigProperties.java

```diff
@@ -79,7 +77,7 @@ public void setSources(List<Source> sources) {
 
 	@Override
 	public String getConfigurationTarget() {
-		return TARGET;
```

Comment:
unless a `String` is repeated in code for readability reasons, it is not needed as a constant in the form of `private static final`